docs/AutomationProtocols.md

```diff
@@ -0,0 +1,98 @@
+---
+id: automationProtocols
+title: Automation Protocols
+---
+
+With WebdriverIO you can choose between multiple automation technologies when running your e2e tests locally or in the cloud. By now almost all modern browser support next to [WebDriver](https://w3c.github.io/webdriver/) also another native interface called [DevTools](https://chromedevtools.github.io/devtools-protocol/) that can be used for automation purposes. Both have advantages and disadvantages depending on your use case and environment.
+
+## WebDriver Protocol
+
+> [WebDriver](https://w3c.github.io/webdriver/) is a remote control interface that enables introspection and control of user agents. It provides a platform- and language-neutral wire protocol as a way for out-of-process programs to remotely instruct the behavior of web browsers.
```

Comment:
I think this should be: `language-neutral protocol` instead of `language-neutral wire protocol`

Reply:
maybe, but it is a quote copied from the spec text